core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.html

```diff
@@ -54,7 +54,9 @@
         </button>
       </td>
       <td class="delete-icon">
-        <span ng-if="canDeleteSkill()" ng-click="deleteSkill(skill.id)" class="fas fa-trash-alt list-summary protractor-test-delete-skill-button"> </span>
+        <a>
+          <span ng-if="canDeleteSkill()" ng-click="deleteSkill(skill.id)" class="fas fa-trash-alt list-summary protractor-test-delete-skill-button"> </span>
```

Comment:
Add the click handler to the anchor tag (also the ng-if and protractor class.)

Reply:
I will do that now, but any specific reason for this?
I saw a code sample where there were ng-if tags in the span so that we can have multiple span tags to correspond to multiple if conditions, rather than multiple anchor tags.

Reply:
Because we are expecting anchor tag as a button and button should handle the click event. (and similar reason for ng-if)

Reply:
Done, please tell me if any other changes are required.
Thanks!